src/components/Card/Card.js

```diff
@@ -1,14 +1,7 @@
 import React from "react";
 import "./Card.css";
 
-const Card = ({
-  id,
-  title,
-  poster_path,
-  release_date,
-  average_rating,
-  showMovieDetails,
-}) => {
+const Card = ({ id, title, poster_path, release_date, average_rating, showMovieDetails}) => {
```

Comment:
Per our in-person conversation, we may revert to the standard convention of wrapping these restructured variables down.  A new chore will be added to the project board.